### ISSUE TITLE
PS Plus Only Launching Installer After Install?

### ISSUE THREAD

**Issue by MEXdave1997** [Hollowscope4022/Playstation-Plus-linux#4]
OK, so after following all the steps to completing the installation, I went ahead to trying an finding where to place the MSVCR file, only to find that PS Plus isn't in my default installation folder, and every time I launch the config, it just launches the installer. Not sure where to go from here.

**Comment by stignarnia** [comment]
Yeah you need to manually go to `/home/<user>/.wine/drive_c/Program Files (x86)/PlayStationPlus`, do the DLL thing, then right click on the game, configure, game options and put `/home/<user>/.wine/drive_c/Program Files (x86)/PlayStationPlus/pspluslauncher.exe` as executable.
